libraries/ms-common/test/test_can_unpack_impl.c

```diff
@@ -0,0 +1,44 @@
+#include "can_unpack_impl.h"
```

Comment:
Can you add some tests for cases where not all values are used?

Reply:
Done. Although, I'm not quite sure what this adds given I added tests with all `NULL` as well?

Reply:
Oh, for the data length.